packages/noco-docs/content/en/engineering/unit-test.md

```diff
@@ -1,41 +1,68 @@
 ---
-title: "Writing Tests"
-description: "Overview to testing"
+title: "Writing Unit Tests"
+description: "Overview to unit testing"
 position: 3250
 category: "Engineering"
-menuTitle: "Tests"
+menuTitle: "Unit tests"
 ---
 
 ## Pre-requisites
+
 - MySQL is preferrable - however we fallback to SQLite
```

Comment:
Pre-requisites is mostly for what things need to be installed on our system. 

CLI commands can come in setup.